convert.py

```diff
@@ -202,6 +254,7 @@ def convert_text_chunks_to_speech(ebook_chunks):
     chunk_mp3_filepaths = []
     for chunk in ebook_chunks:
         if (chunks_converted < MAX_CHUNK_COUNT):
+            chunk_mp3_filepath = None

```

Comment:
Is this necessary?

Reply:
It is not - not sure how it snuck in there!